vmdb/app/models/ems_refresh/parsers/openstack_infra.rb

```diff
@@ -63,6 +63,27 @@ def hosts
         @hosts ||= @baremetal_service.nodes.details
       end
 
+      def cloud_emss
+        @ems.try(:provider).try(:cloud_ems)
```

Comment:
@blomquisg I found the emss in the code already, so I decided to be consistent. :-)

You are probably right, the first try should not be needed